js/Gantt/Customizations.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Customization
-description: Customization
+title: Customization in JavaScript Gantt | Syncfusion
+description: Learn here about customization in Syncfusion Essential JavaScript Gantt Control, its elements, and more.
```

Comment:
Learn here about customization support in Syncfusion Essential JavaScript Gantt Control, its elements, and more.